src/compiler/parser.ts

```diff
@@ -407,9 +410,15 @@ namespace ts {
             case SyntaxKind.JSDocComment:
                 return visitNodes(cbNode, cbNodes, (<JSDoc>node).tags);
             case SyntaxKind.JSDocParameterTag:
-                return visitNode(cbNode, (<JSDocParameterTag>node).preParameterName) ||
-                    visitNode(cbNode, (<JSDocParameterTag>node).typeExpression) ||
-                    visitNode(cbNode, (<JSDocParameterTag>node).postParameterName);
+            case SyntaxKind.JSDocPropertyTag:
+                if ((node as JSDocPropertyLikeTag).isParameterNameFirst) {

```

Comment:
Maybe this should wait for a separate PR, but I would much prefer to see code like:
```ts
const { isParameterNameFirst, fullName, typeExpression } = node as JSDocPropertyLikeTag;
```
in all of these cases, instead of repeated casting.